inductiva/molecules/scenarios/md_water_box/post_processing.py

```diff
@@ -0,0 +1,39 @@
+"Postprocessing steps for the MDWaterBox scenario."
+import os
+import MDAnalysis as mda
+import nglview as nv
+from pathlib import Path
+
+
+class MDWaterBoxSimulationOutput:
+    """Post process the simulation output of a MDWaterBox scenario."""
+
+    def __init__(self, sim_output_path: Path = None):
+        """Initializes a `MDWaterBoxSimulationOutput` object.
+
+        Given a simulation output directory that contains the standard files
+        generated by a MDWaterBox simulation run, this class provides methods to
+        visualize the simulation outputs in a notebook interactively.
+
+        Args:
+            sim_output_path: Path to the simulation output directory."""
+
+        self.sim_output_dir = sim_output_path
+
+    def render(self):
+        """Visualize the simulation outputs in a notebook using NGLView."""
+
+        topology = os.path.join(self.sim_output_dir, "eql.tpr")
+        trajectory = os.path.join(self.sim_output_dir, "eql.xtc")
+        universe = mda.Universe(topology, trajectory, all_coordinates=True)
+        atoms = universe.atoms
+        transformation = mda.transformations.unwrap(atoms)
+        universe.trajectory.add_transformations(transformation)
+
+        view = nv.show_mdanalysis(universe)
+        view.add_ball_and_stick("all")
+        view.center()
+        view.parameters = {
+            "backgroundColor": "white"
```

Comment:
This should be configurable.

Reply:
the background color? I don't think it is that of an important parameter. But we can pass a dict with the view parameters maybe, that would give more flexibility to the user.

Reply:
It might be for design presentation reasons. 

What do you think are the parameters in the view_parameters users should have available?